doc/bus.rst

```diff
@@ -59,6 +59,14 @@ Filtering
 Message filtering can be set up for each bus. Where the interface supports it, this is carried
 out in the hardware or kernel layer - not in Python.
 
+Example defining two filters, one to pass 11-bit ID 0x451, the other to pass 29-bit ID 0xA0000:
+
+.. code-block:: python
+
+    filters = [{"can_id": 0x451, "can_mask":0x7FF,"extended":False},{"can_id": 0xA0000, "can_mask":0x1FFFFFFF,"extended":True}]
```

Comment:
Please make sure that the code conforms with pep8 and black formatting.
See https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements